docs/user/security/authorization/index.asciidoc

```diff
@@ -75,6 +75,26 @@ NOTE: {kib} automatically surrounds your DLS query with a `query` block, so you
 [role="screenshot"]
 image::security/images/create-role-dls-example.png[Create role with DLS index privileges]
 
+[[adding_remote_index_privileges]]
+==== Remote index privileges
+
+If you have at least a platinum license, you can manage access to indices in remote clusters. 
+
+You can assign the same privileges, document-level, and field-level as for <<adding_index_privileges,local index privileges>>. 
+
+[[remote_index_privilege_example_1]]
+===== Example: Grant access to indices in remote clusters
+
+. Go to **Stack Management > Roles**, and then click **Create role**.
+. In **Remote index privileges**, enter:
+.. The name of your remote cluster in the **Remote clusters** field.
```

Comment:
Nit: would it make sense to cross-link to the docs for configuring remote clusters?